escher/scene/model.h

```diff
@@ -4,29 +4,32 @@
 
 #pragma once
 
-#include <memory>
 #include <vector>
 
+#include "escher/base/macros.h"
 #include "escher/scene/object.h"
 
 namespace escher {
 
+// The model to render.
+//
+// TODO(jeffbrown): This currently only contains a vector of objects to be
+// rendered but later on we may store additional resources used to
+// interpret the contents of the model (such as tunable parameters).
 class Model {
  public:
   Model();
-  explicit Model(std::vector<std::unique_ptr<Object>> objects);
+  explicit Model(std::vector<Object> objects);
```

Comment:
This implies that Object, which might not be what we want.  That would imply that any heavy resources referenced by Objects would need to be ref counted.